### PR TITLE
Rename files from "phonemic"/"phonetic" to "broad"/"narrow"

### LINKED ISSUE #389: Phonemic/phonetic vs. broad/narrow
I think that we should reconsider the naming conventions for our data, change "phonemic" and "phonetic" to "broad" and "narrow," respectively. First, I think it would more accurately represent the nature of our data. Second, it would reduce the possibility of mixing up the two types of data. Third, I think the "broad/narrow" distinction is more transparent to non-linguists than "phonemic/phonetic" (since the "broad" and "narrow" are used to refer to level of detail in non-technical vocabulary, e.g. "broadly speaking," "narrowly defined") Finally, you would get the nice bonus of autofilling with the tab key in the command line much sooner.

### PR DESCRIPTION
**Pull request by ajmalanoski**
- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

Closes #389 

I renamed the files with this script: [link](https://gist.github.com/ajmalanoski/6896859b9e9b57ad183e772a74135fae)

I only changed the filenames, so I'm sure a lot of stuff is broken at the moment…

### CODE REVIEW

**Comment by kylebgorman** [comment]
Is your plan for this PR that you'll also make this change to the scrape and testing code?

**Comment by ajmalanoski** [comment]
Yeah. At this point the pull request is still a draft

**Comment by ajmalanoski** [comment]
The name change sort of involved some degree of reconceptualization of the phone-listing process, as expressed in `data/phones/HOWTO.md`. I edited the file in a way that I thought made sense, but let me know what you think.

File: data/phones/HOWTO.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ file:
...
-Finally, for phonemic lists, there may be segments that are properly considered
-pure allophones but appear in phonemic transcriptions. However, such segments
-may be quite frequent in the data and removing all pronunciations that contain
-them would greatly reduce the amount of available data. Therefore, we prefer to
-simply add a comment of the form `# Allophone of ...`; such annotations will
-ultimately be used improve Wiktionary itself.
+When creating a `.phones` file for broadly transcribed data, the goal is often
+to create something that approximates a list of phonemes. However, there may 
+be segments that are properly considered pure allophones but appear in broad 
+transcriptions. However, such  segments may be quite frequent in the data and
```
Comment (kylebgorman) [inline data/phones/HOWTO.md:22]: I like what you wrote here but very small nit-pick: there are two spaces after the word "such"

**Review by kylebgorman — APPROVED** [review]
All looks good to me. Still a failing black test, which should be easy to fix.

**Comment by ajmalanoski** [comment]
I didn't change `tests/test_wikipron/test_config` or `tests/test_wikipron/test_scrape` because I was confused by the code. I changed `wikipron/cli.py`, but I'm not really sure what that script is for, so someone should check that…

**Comment by kylebgorman** [comment]
`cli.py` controls the command-line `wikipron` (which perhaps you've never used but it works nicely). The change you made there is fine but you're effectively renaming `args.phonetic` (or something like that), but you have to rename it everywhere in that directory, not just in `cli.py`.

**Comment by lfashby** [comment]
Hey, thanks for taking this on. The annoying part of this is now going to be going through the `test_wikipron`, `wikipron` and `data` directories to make sure everything still works. In particular fixing `scrape/lib/generate_tsv_summary.py` could be a challenge. Once all the tests pass it might be a good idea to run a restricted big scrape of a small language, say Basque (and maybe a small language with a dialect like Armenian?), and run the postprocessing code just to make sure that all the files are being generated correctly.

**Comment by ajmalanoski** [comment]
I think I'm going to need some help with the stuff in `wikipron/` and `tests/`. Can we make a new branch so that other people can make changes too?

**Comment by kylebgorman** [comment]
Sure, you can issue a PR against a non-master/main branch.

**Comment by ajmalanoski** [comment]
I'm not sure what went wrong with the CircleCI tests/how to fix the problem. Any help?

**Comment by lfashby** [comment]
Hey, I'll take a more through look at this tomorrow but for now changing this [line](https://github.com/kylebgorman/wikipron/blob/27b73e262551402e55ea5bb7c0c33e6aa47a4ec0/data/scrape/scrape.py#L93) ought to resolve the current problem.

**Comment by ajmalanoski** [comment]
Thanks for catching that, Lucas!

**Comment by ajmalanoski** [comment]
Oh yay, all the tests are finally passing

**Review by kylebgorman — APPROVED** [review]
Submit away

**Pull request merged**